nixos/doc/manual/configuration/sshfs-file-systems.section.md

```diff
@@ -0,0 +1,111 @@
+# SSHFS File Systems {#sec-sshfs-file-systems}
+
+[SSHFS](sshfs) is a [FUSE](fuse) filesystem that allows easy access to
+directories on a remote machine using the SSH File Transfer Protocol (SFTP).
+It means that if you have SSH access to a machine, no additional setup is
+needed to mount a directory.
```

Comment:
While it is currently being debated at this issue https://github.com/NixOS/nixpkgs/issues/105243, it seems like the current consensus is that we should not use hard line breaks within paragraphs.

Reply:
Ok, I'll reformat. I guess it will keep the diffs readable but It's kind of contrary to the philosophy of markdown, though: the plaintext should be as readable as the rendered document.

Reply:
Done.